src/components/ActivityTimeline/index.d.ts

```diff
@@ -3,10 +3,14 @@ import { BaseProps } from '../types';
 
 type Names = string[] | string;
 
+interface onToggleSectionEventShape {
+    name: Names;
+}
+
 export interface ActivityTimelineProps extends BaseProps {
     children?: ReactNode;
     multiple?: boolean;
-    onToggleSection?: (event: MouseEvent<HTMLElement>, name: Names) => void;
+    onToggleSection?: (value: onToggleSectionEventShape) => void;
```

Comment:
we rethink this, and the use case we have is to make a fetch when opening a section, then if here we get an array with all section opened we don't know which was toggled, then we will change this API to:
```({ activeSectionNames: Names, toggledSection: string }) => void;```